src/guardian/keys-validation/keys-validation.service.ts

```diff
@@ -27,29 +43,63 @@ export class KeysValidationService {
   ): Promise<{ key: string; depositSignature: string }[]> {
     const forkVersion: Uint8Array = await this.forkVersion();
 
-    const validatedKeys: [
-      {
-        key: string;
-        depositSignature: string;
-        used: boolean;
-      },
-      boolean,
-    ][] = await this.keyValidator.validateKeys(
-      vettedKeys.map((key) => ({
-        key: key.key,
-        depositSignature: key.depositSignature,
-        used: false,
-        withdrawalCredentials: bufferFromHexString(withdrawalCredentials),
-        genesisForkVersion: Buffer.from(forkVersion.buffer),
-      })),
-    );
+    const keysForValidation = vettedKeys
+      .map((key) =>
```

Comment:
`.filter(this.filterKeyForValidation).map((key)=>this.depositData(key, withdrawalCredentials, forkVersion))`

Reply:
filterKeyForValidation - removes all validated keys
depositData - converts all keys to format that is ready for validation

Reply:
fixed